Jenkinsfile

```diff
@@ -66,7 +66,7 @@ def runRestoreMonitoringStack(){
 pipeline {
     agent {
         label {
-            label "sct-builders"
+            label "aws-sct-builders-eu-west-1-v2-CI"
```

Comment:
Maybe a more meaningful name?
When I back to these names later in Jenkins, I don't remember what they means and if they are the correct ones and need to check the code.

Reply:
the name now tell you it's in aws, which region it is, and that it's used for CI.
what more meaning you are missing here ?